xnotify.c

```diff
@@ -931,7 +931,7 @@ additem(struct Queue *queue, struct Itemspec *itemspec)
 			if(config.vertical) {
 				item->textw = queue->w - config.padding_pixels * 2;
 			} else {
-				item->textw = queue->w - config.image_pixels - config.padding_pixels * 3;
+				item->textw = MAX(1, queue->w - config.image_pixels - config.padding_pixels * 3);
```

Comment:
I noticed this issue, which was already there on master. Basically, if you specify an image wider than the width of the notification, `item->textw` goes negative, and `XCreatePixmap` fails. I work around this by adding an extra 1 pixel for this scenario, but feel free to suggest a more elegant solution!